test/commands/data/import/bulk.nut.ts

```diff
@@ -45,6 +45,21 @@ describe('data import bulk NUTs', () => {
     expect(result.failedRecords).to.equal(0);
   });
 
+  it('should import account records in csv format with PIPE delimiter', async () => {
+    const csvFile = await generateAccountsCsv(session.dir, 'PIPE');
+
+    const result = execCmd<DataImportBulkResult>(
+      `data import bulk --file ${csvFile} --sobject Account --wait 10 --column-delimiter PIPE --json`,
+      { ensureExitCode: 0 }
+    ).jsonOutput?.result as DataImportBulkResult;
```

Comment:
is `as DataImportBulkResult` just there to remove the `| undefined`? If so, why not use optional chaining instead of using an assertion?

Reply:
I think I started doing the assertion in other NUTs and just carried the assertion, I can remove it.